src/server/db/index.js

```diff
@@ -120,7 +120,7 @@ async function generatePasscode (email) {
   }
   const passcode = v4()
   const used = false
-  const expiryMinutes = 30
+  const expiryMinutes = process.env.LOGIN_EXPIRY_MINUTES || 30
```

Comment:
I think you need a parseInt() around the environment variable so you will get addition instead of concatentation in line 125 below

Reply:
I tried these from the command  line:

```
EXPIRY_MINUTES=20 node -e 'd = new Date().getMinutes() + process.env.EXPIRY_MINUTES; console.log(d)'

EXPIRY_MINUTES=20 node -e 'd = new Date().getMinutes() + parseInt(process.env.EXPIRY_MINUTES); console.log(d)'
```

And they return different results